api/provider/aws/admin.go

```diff
@@ -0,0 +1,61 @@
+package aws
+
+import (
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/quintilesims/layer0/api/tag"
+	awsc "github.com/quintilesims/layer0/common/aws"
+	"github.com/quintilesims/layer0/common/config"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+type AdminProvider struct {
+	AWS      *awsc.Client
+	TagStore tag.Store
+	Config   config.APIConfig
+}
+
+func NewAdminProvider(a *awsc.Client, t tag.Store, c config.APIConfig) *AdminProvider {
+	return &AdminProvider{
+		AWS:      a,
+		TagStore: t,
+		Config:   c,
+	}
+}
+
+func (a *AdminProvider) Init() error {
+	log.Printf("[DEBUG] Adding API tags")
+
+	fqAPI := addLayer0Prefix(a.Config.Instance(), "api")
+	service, err := readService(a.AWS.ECS, fqAPI, fqAPI)
+	if err != nil {
+		return err
+	}
+
+	// arn format: "arn:aws:ecs:region:123:task-definition/l0-instance-api:1"
+	taskDefinitionARN := aws.StringValue(service.TaskDefinition)
+	split := strings.Split(taskDefinitionARN, ":")
+	taskDefinitionRevision := split[len(split)-1]
+
+	tags := []models.Tag{
```

Comment:
How did we add these tags previously? I don't recall this existing in `admin`...

Reply:
`api/logic/admin_logic`